src/components/molecules/project.tsx

```diff
@@ -98,7 +117,7 @@ const ProjectSettings = ({ session }: IProjectSettingsProps) => {
                     <Image
                       src={team.image && team.image.downloadUrl}
                       fallbackSrc="https://media.graphcms.com/yT9VU4rQPKrzu7h7cqJe"
-                      alt={`${team.name}'s organization image`}
```

Comment:
I was under the impression that you shouldn't leave `alt`s blank... what benefit does it have to remove this?